packages/fxa-shared/package.json

```diff
@@ -78,6 +78,7 @@
   },
   "dependencies": {
     "@fluent/langneg": "^0.6.1",
+    "@hapi/joi": "15.1.1",
```

Comment:
Q: Why adding @hapi/joi back here?

Reply:
Without it, a warning appears after `yarn install` since the `typesafe-joi` dependency is a fork of @hapi/joi v15.x.
(Similar to the one in my comment regarding fxa-support-panel)

Reply:
I hate to say this, because it means more refactoring, but I really question the use of 'typesafe-joi' (https://www.npmjs.com/package/typesafe-joi). There are only 899 weekly downloads, the author states they don't have time to maintain it, and the package hasn't been updated in two years.

Reply:
@bbangert had mentioned in a thread that `typesafe-joi` was used to create some validation and type integrity in a few spots, but it would need some other solution now.

I think if we remove`typesafe-joi`, it would be in its own separate ticket.

Reply:
I just looked to into this further. All I see being used that is special to typesafe-joi is the use of joi.Literal, and really it'd be trivial to to just declare these types manually as it's not used extensively. Maybe bring this up with @bbanger, but considering the package is no longer maintained I don't see the advantage to keeping it.